src/ImageSequencer.js

```diff
@@ -232,9 +235,12 @@ ImageSequencer = function ImageSequencer(options) {
       str.substr(0,str.indexOf('(')),
       str.substr(str.indexOf('(')+1)
     ]
-    
+    str[1] = str[1].split('%20').join(' ');
```

Comment:
Would this then be `decodeURIComponent()` to reconstruct it? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/decodeURIComponent

I had mistakenly put `%20` which is the code for a space, not a comma. I think commas are `%2C` but these converter functions should deal with it.